### DISCUSSION TITLE
XML tartalma - egyedi esetekkel

### DISCUSSION THREAD

**Discussion by Szildiko** [nav-gov-hu/eVAT#31] [Kérdések és válaszok - Gépi interfész] [answered]
Tisztelt Illetékesek!

1) Ha a partnerem Private person státusszal van, akkor akár csak a számlabeküldésnél itt se kell a partner adatokat szerepeltetni?

2a)  Saját kis rendszerünkkel dolgoztam, abban vagyok képben, abban pörögnek a gondolataim. Most próbálom a NAV gondolatával összevetni a működésünket és ez azért nem könnyű. Ha jól értem a dokumentációt:
"Egy áfaanalitika sor egy bizonylathoz kapcsolódik, melyen belül vannak az adott bizonylat standard és vállalati adókódjai."
Van egy Telekom számla:
o	Internet 5%
o	Telefon 27% 
o	Parkolás teljes tiltás, de a számlán szereplő áfa összeg 27%
o	Készülékbiztosítás díja adómentes
Eddig a mi rendszerünkben az áfa bevallásnak megfelelő sorrendben fel voltak sorolva a sorok és hozzájuk voltak rendelve a számla tételek. Tehát kezdtük az 5%-os kulcsokkal és alatta felsoroltuk az összes számla 5%-os tételét aztán volt egy összesen sor. Majd jött 27%-os sor felsoroltuk az összes számla olyan tételét.
Az eÁFA rendszer viszont nem így várja el az adatok szerepeltetését, hanem hozom az első számlát és arról minden információt jelentek (taxinformation elem annyiszor fog szerepelni ahány standardtaxcode-t érint)? 
Nem kell semmilyen összesítés, hogy az adott soron/standardadó kódon mennyi összeg szerepel az adott bevallási időszakban? (lehet hogy hiányosan értelmeztem a dokumentációkat).
Először úgy gondoltam, hogy attól hogy ez egy XML lesz, még a felhasználónak lehetne egy nyomtatási pdf-et adni amin ő is el tudna igazodni, eÁFA analitika pdf, hogy mi lett jelentve, de egyelőre úgy érzem, hogy ez egy nehezen átlátható nyomtatási előkép lesz ezek alapján.

2b) Valamint a parkolás teljes tiltás, nem szerepel a saját rendszerünkben az áfa analitikán, hiszen nem része a 65s bevallásnak, most viszont az eÁFA analitika elvárja, hogy szerepeltessem? (feltételezem, azért mert nincs m lap, attól még kell egyeztetésre?)

2c) Hogyan kell jelenteni a készülékbiztosítás díját, ami adómentes?

2d) Az áfaanalitikában továbbra se kell szerepeltetni azokat a költség számlákat, amiből 1 ft se kerül levonásra? Pl Repi, üzemanyag számlák? (hasonló logika mint az M lapnál?)

2e) Tegyük fel, hogy a Telekom számlát egy pénzforgalmis cég könyveli. Eddig az M lap úgy szólt, hogy első kiegyenlítéskor a számla teljes értékét kell jelentenem, aztán semmit. eÁFÁnál már nincs M lap, tehát ahány kiegyenlítés annyiszor szerepel a számla, ilyenkor lehet 1-2 forint eltérés a részek összeadásánál és online számla által jelentett adatok között. Ilyen jellegű ellenőrzést fog végezni a rendszer, ha igen, akkor lesz-e benne tűrés határ?

2f) A mi rendszerünk például egy ilyen számlára készülék biztosítás arányára kiszámolta hogy nettó 471 ft és áfa összeg -1, f eltételezem egy kerekítési korrekciót így próbált megoldani, ha ez kerülne jelentésre az eÁFÁ-nál nem tudom mi történne. Eddig ugye ezer forintra ment minden, ráadásul ez nem áfás tétel, így az ilyen számításbeli korrekcióknak nem volt jelentőségük, nem tudom most lesz-e?

3. A saját rendszerünk se tökéletes, de senkit nem zavart a következő (és mindig volt más teendő amivel foglalkozzunk). Ha egy pénzforgalmis számla pl EUR-ban van kiállítva és ezt kifizetik teljes összegben, amikor a kiegyenlített tételre árfolyamkülönbözet számolódik, akkor  programunk készít egy nullás összegű generált bizonylatot, mert kiegyenlítéskor az áfát a rendszer már letudta, de az árfolyamot is kiegyenlítésnek ítéli, ezért a generálás arra is lefut. Baj-e ha ezek a nullás bizonylatok is bekerülnének az eÁFA analitikába?

4. Költség számla helyesbítő számlája, de a helyesbítőn nincs tételsor, mert cím vagy akár megjegyzés miatt kerül kiállításra, tehát egy nullás helyesbítő számla, ennek meg kéne jelennie az eÁFA analitikában vagy nem?

5. Ügyfél eset, hogy számlákat importál. Egy számlán több cikk szerepel. Az ügyfél nem összesen adattal importál, hanem a részletes cikkeket, tehát ha egy számlán 200 cikk van, akkor ott 200 áfa sor keletkezik. A mi analitikánkon ez 200 sor is. Az XML-be össze kell vonni ezeket és összesen adatot jelenteni, vagy mehet ahogy rögzítve van?

6. Egyszeres programunkban, egy gazdasági esemény 1 könyvelési bizonylat. Ha van egy metró számla, ami tartalmaz tisztítószereket, irodaszereket és repit, akkor ezt az ügyfél 3 könyvelési bizonylatra könyvelheti, ha precízen akarja vezetni a költségnyilvántartását. Ezekre van egy jelölő, ami alapján az M lapra összefűzi az adatokat. Az XML-ben egy számlasorszáma szerepelhet többször, lehetnek külön jelentve az adatok, vagy az XML-re is össze kell fűzni őket, mint az M lapra? Értékesítés oldalon nincs jelölő, ott külön vannak ezek a könyvelési bizonylatok. 
"Tehát az áfaanalitika bizonylatra, bizonylat azonosítóra és nem adókódra épül." A lineNumber az a számlát jelenti vagy a könyvelési bizonylatot?
Egyszeresben mi nem 467 és 466 kontírokat használunk, hanem 191 fizetendő áfa, 213 levonható áfa, probléma ha ilyen glAccountId kerül jelentésre? 

Előre is köszönök minden segítséget!

**Comment by NTCA-HM** [comment, marked as answer]
Kedves @Szildiko !

A szövegben feltett kérdésekre a válaszok az alábbiak:

1. Ha a partnerem Private person státusszal van, akkor akár csak a számlabeküldésnél itt se kell a partner adatokat szerepeltetni?
Az XSD szerint az adatok megadása nem kötelező. Magán személyek esetén nem kell tölteni.
2a) Saját kis rendszerünkkel dolgoztam, abban vagyok képben, abban pörögnek a gondolataim. Most próbálom a NAV gondolatával összevetni a működésünket és ez azért nem könnyű. Ha jól értem a dokumentációt:
"Egy áfaanalitika sor egy bizonylathoz kapcsolódik, melyen belül vannak az adott bizonylat standard és vállalati adókódjai."
Van egy Telekom számla:
o Internet 5%
o Telefon 27%
o Parkolás teljes tiltás, de a számlán szereplő áfa összeg 27%
o Készülékbiztosítás díja adómentes
Eddig a mi rendszerünkben az áfa bevallásnak megfelelő sorrendben fel voltak sorolva a sorok és hozzájuk voltak rendelve a számla tételek. Tehát kezdtük az 5%-os kulcsokkal és alatta felsoroltuk az összes számla 5%-os tételét aztán volt egy összesen sor. Majd jött 27%-os sor felsoroltuk az összes számla olyan tételét.
Az eÁFA rendszer viszont nem így várja el az adatok szerepeltetését, hanem hozom az első számlát és arról minden információt jelentek (taxinformation elem annyiszor fog szerepelni ahány standardtaxcode-t érint)?

_Válasz: 
Igen_

Nem kell semmilyen összesítés, hogy az adott soron/standardadó kódon mennyi összeg szerepel az adott bevallási időszakban? (lehet hogy hiányosan értelmeztem a dokumentációkat).

_Válasz:
Nem kell, tekintettel arra, hogy a tételes adatokból ez számolható. Az analitikában pedig meg kell az adózónak adnia a „Végleges számokat”. A tételekből ez ellenőrizhető._

Először úgy gondoltam, hogy attól hogy ez egy XML lesz, még a felhasználónak lehetne egy nyomtatási pdf-et adni amin ő is el tudna igazodni, eÁFA analitika pdf, hogy mi lett jelentve, de egyelőre úgy érzem, hogy ez egy nehezen átlátható nyomtatási előkép lesz ezek alapján.

_Válasz:
Az adózó a saját adatait a programban úgy jeleníti meg ahogy szeretné, nem az XML-ből kell ezeket az adatokat visszamutatni, hiszen azt a saját adataiból képezi le._

2b) Valamint a parkolás teljes tiltás, nem szerepel a saját rendszerünkben az áfa analitikán, hiszen nem része a 65s bevallásnak, most viszont az eÁFA analitika elvárja, hogy szerepeltessem? (feltételezem, azért mert nincs m lap, attól még kell egyeztetésre?)

_Válasz: 
Igen, rendelkezni kell a számla minden tételéről._

2c) Hogyan kell jelenteni a készülékbiztosítás díját, ami adómentes?

_Válasz:
Az erre vonatkozó adókóddal._

2d) Az áfaanalitikában továbbra se kell szerepeltetni azokat a költség számlákat, amiből 1 ft se kerül levonásra? Pl Repi, üzemanyag számlák? (hasonló logika mint az M lapnál?)

_Válasz:
Nem kell szerepeltetni, de ha szeretné valaki akkor belerakhatjam természetesen 0 Forinttal._

2e) Tegyük fel, hogy a Telekom számlát egy pénzforgalmis cég könyveli. Eddig az M lap úgy szólt, hogy első kiegyenlítéskor a számla teljes értékét kell jelentenem, aztán semmit. eÁFÁnál már nincs M lap, tehát ahány kiegyenlítés annyiszor szerepel a számla, ilyenkor lehet 1-2 forint eltérés a részek összeadásánál és online számla által jelentett adatok között. Ilyen jellegű ellenőrzést fog végezni a rendszer, ha igen, akkor lesz-e benne tűrés határ?

_Válasz:
Alapvetően bevallási sorokat 1000 ft-ra kerekítünk, tehát valószínűleg 1-2 ft nem okoz gondot.  
Pénzforgalmisnál, igen ahány kiegyenlítés annyiszor kell szerepeljen.
Ellenőrzési szempontok a specifikációban láthatók._

2f) A mi rendszerünk például egy ilyen számlára készülék biztosítás arányára kiszámolta hogy nettó 471 ft és áfa összeg -1, f eltételezem egy kerekítési korrekciót így próbált megoldani, ha ez kerülne jelentésre az eÁFÁ-nál nem tudom mi történne. Eddig ugye ezer forintra ment minden, ráadásul ez nem áfás tétel, így az ilyen számításbeli korrekcióknak nem volt jelentőségük, nem tudom most lesz-e?

_Válasz:
A bevallásban mínusz, 0 ft-nál kevesebb nem adható meg így ezt majd meg kell nézni._ 

A saját rendszerünk se tökéletes, de senkit nem zavart a következő (és mindig volt más teendő amivel foglalkozzunk). Ha egy pénzforgalmis számla pl EUR-ban van kiállítva és ezt kifizetik teljes összegben, amikor a kiegyenlített tételre árfolyamkülönbözet számolódik, akkor programunk készít egy nullás összegű generált bizonylatot, mert kiegyenlítéskor az áfát a rendszer már letudta, de az árfolyamot is kiegyenlítésnek ítéli, ezért a generálás arra is lefut. Baj-e ha ezek a nullás bizonylatok is bekerülnének az eÁFA analitikába?

_Válasz:
Ez egy saját belső bizonylat, bele lehet tenni, de egyébként sincs ÁFA vonzata._

Költség számla helyesbítő számlája, de a helyesbítőn nincs tételsor, mert cím vagy akár megjegyzés miatt kerül kiállításra, tehát egy nullás helyesbítő számla, ennek meg kéne jelennie az eÁFA analitikában vagy nem?

_Válasz:
Nem kell megjelennie, Nincs megtiltva, bele lehet tenni azonban nincs értelme._

Ügyfél eset, hogy számlákat importál. Egy számlán több cikk szerepel. Az ügyfél nem összesen adattal importál, hanem a részletes cikkeket, tehát ha egy számlán 200 cikk van, akkor ott 200 áfa sor keletkezik. A mi analitikánkon ez 200 sor is. Az XML-be össze kell vonni ezeket és összesen adatot jelenteni, vagy mehet ahogy rögzítve van?

_Válasz: 
Adókódonként összesíthető (ez a minimum elvárás), de 200 sorban is jelenthető, nyilván az előbbi a támogatott._

Egyszeres programunkban, egy gazdasági esemény 1 könyvelési bizonylat. Ha van egy metró számla, ami tartalmaz tisztítószereket, irodaszereket és repit, akkor ezt az ügyfél 3 könyvelési bizonylatra könyvelheti, ha precízen akarja vezetni a költségnyilvántartását. Ezekre van egy jelölő, ami alapján az M lapra összefűzi az adatokat. Az XML-ben egy számlasorszáma szerepelhet többször, lehetnek külön jelentve az adatok, vagy az XML-re is össze kell fűzni őket, mint az M lapra? Értékesítés oldalon nincs jelölő, ott külön vannak ezek a könyvelési bizonylatok.

_Válasz:
Ez ugyan az az eset mint előzőleg a tételeket megfelelően kell adókódolni és szükség szerint összesíteni._

"Tehát az áfaanalitika bizonylatra, bizonylat azonosítóra és nem adókódra épül." A lineNumber az a számlát jelenti vagy a könyvelési bizonylatot?

_Válasz:
Ha az analitikában van 5 db bizonylat, akkor a lineNumber 1-5-ig tart a bizonylatra vonatkozóan. Ezen belül kell megbontani az információt a bizonylatok tekintetében._ 

Egyszeresben mi nem 467 és 466 kontírokat használunk, hanem 191 fizetendő áfa, 213 levonható áfa, probléma ha ilyen glAccountId kerül jelentésre?

_Válasz:
Igen_
